packages/main/src/MessageStrip.ts

```diff
@@ -73,12 +74,10 @@ type DesignTypeAnnouncemnt = Record<MessageStripDesign, string>;
  * <code>import "@ui5/webcomponents/dist/MessageStrip";</code>
  *
  * @constructor
- * @author SAP SE
- * @alias sap.ui.webc.main.MessageStrip
- * @extends sap.ui.webc.base.UI5Element
- * @tagname ui5-message-strip
+ * @extends UI5Element
  * @public
  * @since 0.9.0
+ * @slot {Node[]} default - Defines the text of the component. <br><br><b>Note:</b> Although this slot accepts HTML Elements, it is strongly recommended that you only use text in order to preserve the intended design.
```

Comment:
slot tag supports multiline comments so you can put that description as it was:

```ts
 * @slot {Node[]} default
 * Defines the text of the component.
 * <br><br>
 * <b>Note:</b> Although this slot accepts HTML Elements, it is strongly recommended that you only use text in order to preserve the intended design.
 */
```

Reply:
Done